cli/src/commands/invite/greet.rs

```diff
@@ -61,6 +61,10 @@ pub async fn main(args: Args) -> anyhow::Result<()> {
             let ctx = step4_device(ctx).await?;
             step5_device(ctx).await
         }
+        InviteListItem::ShamirRecovery { .. } => {
+            // See: https://github.com/Scille/parsec-cloud/issues/8841
```

Comment:
```suggestion
            // TODO: https://github.com/Scille/parsec-cloud/issues/8841
```

I'm using a vscode extension that hightlight todo comment if they follow a specific pattern